client/ionic/src/pages/About.tsx

```diff
@@ -44,14 +44,13 @@ const About: React.FC = () => {
                         </IonCardHeader>
                         <IonCardContent className="tl">
                           {screen.bodyTexts &&
-                            screen.bodyTexts.map(txt => <p>{txt}</p>)}
+                            screen.bodyTexts.map((txt, key) => (
+                              <p key={key}>{txt}</p>
```

Comment:
What is the key used for in styling?

Reply:
The key is just a react thing, sort of like an index for array elements, it doesn't affect styling.

Reply:
I was just sorting out the jsx error while I was at it. elements within a map require a key